qa/test_template.sh

```diff
@@ -15,7 +15,7 @@ apt-get install -y --no-install-recommends libsm6 libice6 libxrender1 libxext6 l
 # to make sure defaults are right
 apt-get install -y --no-install-recommends --reinstall python$PYVER python$PYVER-dev
 
-CUDA_VERSION=$(nvcc --version | grep -E ".*release ([0-9]+)\.([0-9]+).*" | sed 's/.*release \([0-9]\+\)\.\([0-9]\+\).*/\1\2/')
+CUDA_VERSION=$(cat /usr/local/cuda/version.txt | sed 's/.*Version \([0-9]\+\)\.\([0-9]\+\).*/\1\2/')
```

Comment:
This line may be used in other scripts as well so please change it there.

Reply:
done